### DISCUSSION TITLE
Can't see my libraries in Kodi with PKC

### DISCUSSION THREAD

**Discussion by GizmoBzh** [croneter/PlexKodiConnect#1813] [Q&A] [answered]
I have a plex server with multiple libraries (different folders for TV shows, movies, mangas...) and are visible independantly in Plex. But once in Kodi with PKC everything is mixed in the movies and TV shows entries...

Possible to have the different libraries visible in kodi ?

**Comment by croneter** [comment, marked as answer]
There a x different issues and discussions asking exactly this question. See here how to "solve" this issue: https://github.com/croneter/PlexKodiConnect/wiki/Video-Nodes-and-Skins

**Reply by GizmoBzh** [reply]
Thanks a lot, i searched but bid not found it...
